File: API/Entites/Proizvod.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace API.Entites
{
    public class Proizvod
    {
        public int Id { get; set; }
        public string Ime { get; set; }
        public string Opis { get; set; }
        public string Velicina { get; set; }
        public string Boja { get; set; }
        public long Cena { get; set; }
        public string PictureUrl { get; set; }
        public string Tip { get; set; }
        public string Brend { get; set; }
        public string KolicinaNaStanju { get; set; }


        
    }
}

File: client/src/app/models/proizvod.ts

export interface Proizvod {
    id: number;
    ime: string;
    opis: string;
    velicina: string;
    boja: string;
    cena: number;
    pictureUrl: string;
    tip?: string;
    brend: string;
    kolicinaNaStanju?: string;
    
}

export interface ProductParams {
    orderBy: string;
    searchTerm?: string;
    tip: string[];
    brend: string[];
    pageNumber: number;
    pageSize: number;
}